src/Cli/dotnet/commands/dotnet-run/LocalizableStrings.resx

```diff
@@ -217,4 +217,7 @@ The current {1} is '{2}'.</value>
   <data name="ValueMustBeAnObject" xml:space="preserve">
     <value>The property '{0}' must be an object if it is specified.</value>
   </data>
-</root>
+  <data name="RunCommandProjectAbbreviationDeprecated" xml:space="preserve">
+    <value>WARNING: The abbreviation of -p for --project is deprecated</value>
```

Comment:
nit: Missing "."

Reply:
The deprecation message doesn't match the spec of `The abbreviation of -p for --project is deprecated. Please use --project.`  The latter part might seem redundant but makes it clear that `--project` is still supported and the replacement.

Reply:
Yes, this was originally based on an earlier version of the spec and has now been updated.